.buildkite/publish/docker-compose.yml

```diff
@@ -60,8 +60,8 @@ services:
       - mariadb
       - mssql
       - mongo
```

Comment:
```suggestion
      - mongodb
```

Reply:
No, I didn't rename the client docker image, it's still called mongo